tensorboard/loader_test.py

```diff
@@ -228,7 +228,10 @@ def testBigNumberInAmerica_showsCommas(self):
     self.assertIn(u' 1,024B/s ', self.bars[0])
 
   def testBigNumberInGermany_showsCommas(self):
-    locale.setlocale(locale.LC_ALL, 'de_DE.utf8')
+    try:
+      locale.setlocale(locale.LC_ALL, 'de_DE.utf8')
+    except Exception:
+      return  # internal CI doesn't have this locale for some reason
```

Comment:
Indeed, folks rarely read logs if nothing's awry. They might occasionally be useful though, ie I wonder why this test passes no matter how I change something. Maybe a warning.